apps/web/src/app/(dashboard)/layout.tsx

```diff
@@ -2,21 +2,28 @@ import {
   SidebarInset,
   SidebarProvider,
 } from '@workspace/ui/components/sidebar';
+import { cookies } from 'next/headers';
 
-import { DashboardHeaderClient } from './_clientBoundary/DashboardHeaderClient';
 import { DashboardSidebarClient } from './_clientBoundary/DashboardSidebarClient';
+import { DashboardHeader } from './_components/DashboardHeader';
 
-export default function DashboardLayout({
+export default async function DashboardLayout({
   children,
 }: {
   children: React.ReactNode;
 }) {
+  // 새로고침해도 사이드바 상태를 유지하기 위해 쿠키에서 값을 가져옴
+  const cookieStore = await cookies();
+  const sidebarCookie = cookieStore.get('sidebar_state')?.value;
+  const defaultOpen =
+    sidebarCookie === undefined ? true : sidebarCookie === 'true';
```

Comment:
[제안]
1. 가독성을 좀더 높여보자면.. 아래처럼은 어떨까요?
```suggestion
  const defaultOpen = sidebarCookie === undefined || sidebarCookie === 'true'
```

2. 네이밍을 isDefaultOpen 어떨까요?
boolean 값이라 is~~, has~~ needs~~ 등 붙이면 어떨까 싶네요. (prop 명이랑 다르지만)